polaris-service/src/main/java/org/apache/polaris/service/tracing/TracingFilter.java

```diff
@@ -53,6 +54,7 @@ public class TracingFilter implements Filter {
   private static final Logger LOGGER = LoggerFactory.getLogger(TracingFilter.class);
   private final OpenTelemetry openTelemetry;
 
+  @Inject
```

Comment:
Generally, the CDI spec demands a public no-arg constructor - some frameworks do not require it though.

Reply:
The spec is pretty clear that you can annotate a constructor with its dependencies - https://jakarta.ee/specifications/cdi/3.0/jakarta-cdi-spec-3.0#bean_constructors